blaze/compute/tests/test_sql_compute.py

```diff
@@ -348,11 +348,11 @@ def test_by_summary_clean():
 
 
 def test_by_summary_single_column():
-    expr = by(t.name, n=t.name.count(), biggest=t.name.max())
+    expr = by(t.name, n=t.name.count(), biggest=t.amount.max())
```

Comment:
The intent of this test is to check a case where the child of the by is a single column.  Previously this was `t.name`, now the child will be just `t`.